string-qq.cabal

```
@@ -27,4 +27,4 @@ test-suite string-qq-test
     type:           exitcode-stdio-1.0
     hs-source-dirs: tests
     main-is:        Test.hs
-    build-depends:  base > 3 && < 6, string-qq, HUnit >=1.6 && <1.7, text >=1.2 && <1.3
+    build-depends:  base > 3 && < 6, string-qq, HUnit >=1.6 && <1.7, text >=1.2 && <2.1
```

```suggestion
    build-depends:  base > 3 && < 6, string-qq, HUnit >=1.6 && <1.7, text >=1.2 && <1.3 || >=2.0 && <2.1
```
Otherwise a hypothetical `text-1.3` would be allowed to break backwards-compat.